test/kubernetes/e2e/tests/manifests/profiles/kubernetes-gateway.yaml

```diff
@@ -4,6 +4,8 @@
 
 # Configuration that is Global to Gloo Gateway
 global:
+  glooMtls:
+    enabled: true
```

Comment:
TODO: Revert this

This is done to ensure that all tests work with this setting (especially istio integration tests)

Reply:
Reverted